README.md

```diff
@@ -36,40 +36,21 @@ make image-build
 
 ## Run
 
-Create a Kind cluster with disabled Node CIDRs allocation:
+### In Cluster
 
-```sh
-kind create cluster --config hack/test/kind/kind-cfg.yaml
-```
-
-Install ClusterCIDR CRD and configure node-ipam-controller to use dual mode (See the [examples](examples) folder for
-more examples):
-
-```sh
-kubectl create -f charts/node-ipam-controller/gen/crds/networking.x-k8s.io_clustercidrs.yaml
-kubectl create -f examples/clustercidr-dual.yaml
-```
-
-Run the controller outside the cluster by specifying Kind cluster kubeconfig:
-
-```sh
-./bin/manager --kubeconfig="$HOME"/.kube/config
-```
-
-To run the controller inside the cluster, a Docker image must first be loaded into a registry accessible within the Kind
-cluster.
+The following command runs a kind cluster, builds the controller, and install it in the cluster.
+It also creates a [default ClusterCIDR](./examples/clustercidr-dual.yaml).
 
 ```sh
-docker build -t registry.k8s.io/node-ipam-controller/node-ipam-controller:local -f Dockerfile .
-docker save --output node-ipam-controller.tar registry.k8s.io/node-ipam-controller/node-ipam-controller:local
-kind load docker-image registry.k8s.io/node-ipam-controller/node-ipam-controller:local
+make run
 ```
 
-Check Kind [documentation](https://kind.sigs.k8s.io/docs/user/local-registry/) on how to use local container image
-registry.
+### Outside Cluster
 
-Install node-ipam-controller in the cluster via helm:
+The following command runs a kind cluster and install CRDs.
+NOTE: run the controller with leader election disabled (see `IPAM_ENABLE_LEADER_ELECTION`) or specify `POD_NAME` and
+`POD_NAMESPACE` environment variables that are used as leader election ID and namespace.
 
 ```sh
-helm install node-ipam-controller ./charts/node-ipam-controller --create-namespace --namespace nodeipam --set image.tag=local
```

Comment:
I expect to leave some instructions for users to run their controller in production, this is very developer focused only with `makefiles`